sips/sip-171.md

```diff
@@ -0,0 +1,71 @@
+---
+sip: 171
```

Comment:
@andytcf can you change this to 172? Andre Cronje submitted 171 the same day as this and its easier to ask you to change it if thats okay